### DISCUSSION TITLE
Searching traces

### DISCUSSION THREAD

**Discussion by dinmukhamedm** [lmnr-ai/lmnr#129] [Ideas]
Starting this thread to learn about what general patterns would people use in searching traces and spans.

Some initial questions to get this running:
- Do you (would you like to) have keyword-based search by trace content? E.g. all traces mentioning <company_name>
- Do you (would you like to) have granularity to search for inputs separately from outputs? E.g. all traces where inputs mention <city_name>
- Do you (would you like to) search by semantic similarity or meaning? E.g. all traces that talk about sports or all traces similar to the one I am looking at now

We are curious to learn from you about your use cases and usage patterns, so don't hesitate to share! This will help us direct and guide the development of search here

**Comment by skull8888888** [comment]
we have this!
